hack/quick_install.sh

```diff
@@ -56,6 +56,6 @@ kubectl wait --for=condition=available --timeout=600s deployment/cert-manager-we
 cd ..
 # Install KFServing
 # Retry inorder to handle that it may take a minute or so for the TLS assets required for the webhook to function to be provisioned
-for i in 1 2 3 4 5 ; do kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
+for i in 1 2 3 4 5 ; do kubectl apply -f ../install/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
```

Comment:
This highly depends on where you run the script, I'd suggest to change to use the release yaml `https://github.com/kubeflow/kfserving/releases/download/$KFSERVING_VERSION/kfserving.yaml`